src/Event/DelayedListener.php

```diff
@@ -108,4 +108,12 @@ public function shouldOccur(ModelInterface $model): bool
 
         return false;
     }
+
+    /**
+     * @return callable
+     */
+    public function getListener(): callable
```

Comment:
I'm not sure this is the correct method. If it was not present I think it's better to not have it. To me getting it by reflection for debugging sounds ok.

Another possibility is to add some behavior so the delayed listener is created via factory (o we can create different objects while debugging it "on")

Reply:
You're right. I will use Reflection for this matter